justfile

```diff
@@ -254,9 +254,7 @@ linkerd *flags: _k3d-init
 linkerd-crds-install: _k3d-init
     {{ _linkerd }} install --crds \
         | {{ _kubectl }} apply -f -
-    {{ _kubectl }} wait crd --for condition=established \
-        --selector='linkerd.io/control-plane-ns' \
-        --timeout=1m
+    {{ _linkerd }} check --crds --wait=1m
```

Comment:
I'm -1 on this. The check output is pretty disruptive (spinner wheel etc). Does this solve a problem you hit?

Reply:
Also this wait command _should_ be more robust since it actually checks the CRD status (established) and not just existence in the API

Reply:
I had guessed that I was hitting a race condition where tests were running before the CRDs had been applied, but I'm still hitting the issue with this change.  Will investigate further.